ls1intum/Artemis
# `Exam mode`: Fix exercise statistics page not loading

### Checklist
#### General
#### Client

### Motivation and Context
The exercise statistics page is not loading for exam exercises.

### Description
Instead of `exercise.course`, we should always use the `getCourseFromExercise` wrapper since exam exercises don't have this property set. 

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Exam with at lest 1 text exercise

1. Open the Exam and open the text exercise
2. Click on Exercise Statistics
3. Verify that the page works and that no errors are shown in the console

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests
